src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './style.css';
 
 function App() {
-  return (
```

Comment:
- [ ] Following the project requirements, The `App.js` component should be class-based. Kindly refactor it to a class-based component.